sandbox/host/src/store_refcell.rs

```diff
@@ -83,38 +83,41 @@
 //! Therefore, since it is not possible to pass a reference to Store through nested runtime interface call
 //! or cancel previous mutable borrow, it is necessary to use `StoreRefCell` for safe repeated mutable borrowing of `Store`.
 //!
-
 use std::{
     cell::{Cell, UnsafeCell},
-    num::NonZero,
+    num::NonZeroUsize,
     ops::{Deref, DerefMut},
     ptr::NonNull,
 };
 
 use defer::defer;
-use wasmer::{AsStoreMut, AsStoreRef, Store, StoreRef};
 
 #[derive(Debug, Clone, Copy)]
 enum BorrowState {
-    Shared(NonZero<usize>),
```

Comment:
why? `type NonZeroUsize = NonZero<usize>`

Reply:
I think it makes the code a little bit cleaner.

Reply:
yes, but NonZero<T> is available since 1.79 why not use generic types?

Reply:
I prefer to use non-generic types where possible to avoid overcomplicating things.